packages/tests/tests/store/time_filter.node.spec.ts

```diff
@@ -34,7 +34,7 @@ describe("Waku Store, time filter", function () {
     [-19000, 1, 4],
     [-19000, -2, -1],
     [-19000, 0, 1000],
-    [-19000, -1000, 0],
+    [-19000, -1000, 1], // Changed from 0 to 1 to include the message timestamp
```

Comment:
why changing this test? is it for `Store` and should not affect `LightPush`

Reply:
this specific test was failing locally/ci for me, seems ok now, reverted